ush/parsing_model_configure_FV3.sh

```diff
@@ -14,7 +14,6 @@ FV3_model_configure(){
 
 rm -f model_configure
 cat >> model_configure <<EOF
-print_esmf:              ${print_esmf:-.true.} 
```

Comment:
Why was this removed?

Reply:
Because it is no longer a model_configure variable.  Please see https://github.com/JessicaMeixner-NOAA/global-workflow/commit/d67e8b0154ffe6e5530a8956d0c15705ac578991

Reply:
> > There are some things here that have been changed that I don't know why. Also, as always, I have to question whether some of these physics changes (particularly changing to Thompson MP) should be applied to everything or just coupled. Changes like these reverberate out much farther than just the prototypes (with particular danger to GDAS). I know this isn't Jessica specifically, so I need someone from physics to enlighten us and sign off on these @yangfanglin.
> > There may be more, but this is a first pass.
> 
> Walter, we are building a new model. GDAS will be part of this upgrade when DA is ready. I am not sure what the danger you referred to is.

global-workflow needs to continue working for everyone in the meantime. Prototypes are no longer in their own isolated sandbox. So when changes like these come in, I need to know that the changes are safe for everyone. Otherwise, they need to be particular to the coupled or have some other mechanism to keep them from impacting non-prototype development.

Reply:
Okay, I was finally able to uncover the relevant conversation. So now the analogous setting is in nems.configure, but it's currently hard-coded to ESMF_LOGKIND_MULTI. If we are going to hard-code it, I'd prefer it be ESMF_LOGKIND_MULTI_ON_ERROR, but ideally it would be a configurable setting.

Reply:
Can an issue for this be made and an improvement can be added later or is this a requirement for this PR?

Reply:
Later is fine.

Reply:
But at least change it to ESMF_LOGKIND_MULTI_ON_ERROR for now.

Reply:
The setting needs to be added to the other nems.configure files too.

Reply:
Updated all nems.configure as requested.